doc/getting_started/getting_started.rst

```diff
@@ -339,9 +266,38 @@ and then:
 You can run ``zephyr/zephyr.exe --help`` to get a list of available
 options.  See the :ref:`native_posix` document for more information.
 
-This executable can be instrumented like any other Linux process. For ex. with gdb
-or valgrind.
-Note that the native port is currently only tested in Linux.
+This executable can be instrumented using standard tools, such as gdb or
+valgrind.
+
+.. rubric:: Footnotes
+
+.. [#homebrew_user]
+
+   For details, see
+   https://docs.brew.sh/Homebrew-and-Python#note-on-pip-install---user.
+
+.. [#tools_native_posix]
+
+   Usually, the toolchain is a cross-compiler and related tools which are
+   different than the host compilers and other programs available for
+   developing software to run natively on your operating system.
+
+   One exception is when building Zephyr as a host binary to run on a POSIX
+   operating system. In this case, you still need to set up a toolchain, but it
+   will provide host compilers instead of cross compilers. For details on this
+   option, see :ref:`native_posix`.
+
+.. [#board_misnomer]
+
+   This has become something of a misnomer over time. While the target can be
+   -- and often is -- a microprocessor running on its own dedicated hardware
+   board, Zephyr also supports using QEMU to run targets built for other
+   architectures in emulation, targets which produce native host system
+   binaries that implement Zephyr's driver interfaces with POSIX APIs, and even
+   running different Zephyr-based binaries on CPU cores of differing
```

Comment:
```suggestion
   support running different Zephyr-based binaries on CPU cores of differing
```

Reply:
Hrm, I don't think so. The subject here is "Zephyr", as in "Zephyr also supports" on line 294. If this were rewritten as a bullet list, it would look like this:

```
Zephyr also supports:

- using QEMU to run targets build for other architectures in emulation,
- targets which produce native host system binaries that implement Zephyr's driver interfaces with POSIX APIs,
- and even running different Zephyr based binaries [...]
```

This suggested change edit would result in subject verb disagreement in the last bullet ("Zephyr support"). I could say "supports" again, but that seems redundant.

I can rephrase as a bullet list?

Reply:
I misread the long sentence.  You're right - and a bullet list would be clearer :)